minecraft_servers/vamin/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "server_name": "vamin",
+  "nice_name": "Vamin",
+  "direct_ip": "vamin.de",
+  "server_wildcards": [
+    "%.vamin.de",
+    "%.vamin.eu"
+  ],
+  "social": {
+    "web": "https://vamin.de",

```

Comment:
The website is not available. Please remove the line or enter a correct, callable url.
You can still make aother PR later, as soon as the website is online, but we do not want to have invalid data in the repository - Also to prevent dead links ✌

Reply:
I'm sorry my mistake. I have now removed the website